modules/sdk/src/main/scala/org/tessellation/sdk/infrastructure/snapshot/MajorityPeerSelect.scala

```diff
@@ -42,40 +55,38 @@ object MajorityPeerSelect {
     val logger = Slf4jLogger.getLoggerFromClass[F](MajorityPeerSelect.getClass)
 
     def select: F[Peer] = selectPeer
+      .flatTap(details => logger.debug(details.asJson.noSpaces))
```

Comment:
in general we write user-readable messages for debugging. what is the reason to return a json there? there is a show instance on the `FiteredPeerDetails`, so you can use this instance for error message

Reply:
@ryle-goehausen asked that the messages to be easily mined from the logs. @sdavidp suggested that JSON would be easier to mine than straight text. So, we went with JSON.

As for why we have so much data, @ryle-goehausen asked for the fields in the `FilteredPeerDetails` object in order to give insight into the peer that was chosen.

Reply:
it will significantly bloat the log file

Reply:
I think you should log only the most important things

Reply:
I will defer to @ryle-goehausen on what is exactly logged.

Reply:
if you really need such granulity and so much details, you should log it to a separate log file (see how we log http requests or gossip) not to bloat the main log file. otherwise, the log file may quickly become unreadable and hard to debug.

Reply:
I'll look into this.

Reply:
So, I have to modify this file:

https://github.com/Constellation-Labs/tessellation/blob/develop/modules/shared/src/main/resources/logback-included.xml

Reply:
it wasn't a question previously, so didn't answer. yes, exactly